content/post/multimedia/channel/pragma/index.ar.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Pragma قناة
+description: القناة عن صناعة البرمجيات ، والكود النظيف ، وتصميم البرامج، والأدوات.
+slug: pragma
+date: 2024-09-08 00:00:00+0000
+image: pragma.png
```

Comment:
Let's find another cover. The image dimensions will not show correctly.
It should be something like 1024 x 768

Reply:
The closest I can get is 1024x576, after using ```magick``` to modify dimensions of it's Facebook group cover keeping the aspect ratio.

Reply:
I pushed another cover

Reply:
It's good enough, thank you :raised_hands: